cnf-certification-test/performance/suite.go

```diff
@@ -47,33 +47,41 @@ var _ = ginkgo.Describe(common.PerformanceTestKey, func() {
 		testExclusiveCPUPool(&env)
 	})
 
+	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestRtAppNoExecProbes)
+	ginkgo.It(testID, ginkgo.Label(tags...), func() {
+		var guaranteedPodContainersWithExclusiveCPUs = env.GetGuaranteedPodContainersWithExlusiveCPUs()
+		testhelper.SkipIfEmptyAll(ginkgo.Skip, guaranteedPodContainersWithExclusiveCPUs)
+		testRtAppsNoExecProbes(&env, guaranteedPodContainersWithExclusiveCPUs)
+	})
+
+	// Scheduling related tests begins here
 	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestSharedCPUPoolSchedulingPolicy)
 	ginkgo.It(testID, ginkgo.Label(tags...), func() {
+		env.Pods = env.GetPodsWithoutHostPID() // Consider only pods with no access to host machine
```

Comment:
We don't want to overwrite the `env.Pods` as this will affect other tests in the suite.

Reply:
The tests which might have got impacted are executed before scheduling tests.

Reply:
I also think we should not do this